couchbase-entity/src/main/java/com/kaufland/model/field/CblConstantHolder.kt

```diff
@@ -24,9 +24,9 @@ class CblConstantHolder(field: Field) : CblBaseFieldHolder(field.name, field) {
 
     override val fieldType: TypeName = TypeUtil.parseMetaType(typeMirror, isIterable, null)
 
-    override fun interfaceProperty(): PropertySpec {
-
-        return PropertySpec.builder(accessorSuffix(), fieldType, KModifier.PUBLIC).build()
+    override fun interfaceProperty(isOverride: Boolean): PropertySpec {
+        val modifiers = listOfNotNull(KModifier.PUBLIC, KModifier.OVERRIDE.takeIf { isOverride })
```

Comment:
I think in the case of "override", you don't need "public" because kotlin implicitly uses the same visibility as the parent

Reply:
I don't think you need them in the non-override case either since public is Kotlin's default visibility. This seems to work from a small test at least. However, I think specifying it explicitely instead of just assuming it implicitly makes it easier to understand what is going on. Since the unnecessary public modifiers don't end up in the generated entities and wrappers anyways I would leave it as it is. What do you think?

Reply:
Hmm
In the non-override case I agree: adding the public modifier improves readability.
In the override case I am not sure but probably ok as well: Right now it is probably ok, because the library is apparently smart enough to ignore the public-modifier if it is not needed. What would happen if we ever wanted to override a non-public method? It would then become public in the subclass which - I think - is allowed by kotlin: Maybe not what is intended but should not crash (and the use-case is probably quite accademic)

Let us see what Sven thinks, but I would be ok with the current implementation

Reply:
For me it's fine to go with the explicit public.